<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExamControl</class>
 <widget class="QDialog" name="ExamControl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>552</width>
    <height>282</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Проведение теста</string>
  </property>
  <widget class="QTableWidget" name="examList">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>331</width>
     <height>231</height>
    </rect>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>160</number>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <column>
    <property name="text">
     <string>ID</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Дата проведения</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="addStudToExam">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>130</y>
     <width>191</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Допустить студентов к тесту</string>
   </property>
  </widget>
  <widget class="QComboBox" name="subjList">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>40</y>
     <width>191</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="selectExam">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>70</y>
     <width>191</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Начать/продолжить тест</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>100</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Время на тест (мин)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="examTime">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>100</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>45</string>
   </property>
  </widget>
  <widget class="QPushButton" name="viewReport">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>160</y>
     <width>191</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Просмотреть отчет</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
